src/rp2_common/pico_audio_i2s/audio_i2s.c

```diff
@@ -52,10 +52,18 @@ const audio_format_t *audio_i2s_setup(const audio_format_t *intended_audio_forma
     gpio_set_function(config->clock_pin_base, func);
     gpio_set_function(config->clock_pin_base + 1, func);
 
+#if PICO_PIO_VERSION > 0
+    if(config->data_pin >= 32 || config->clock_pin_base + 1 >= 32) {
+        pio_set_gpio_base(audio_pio, 16);
+    }
+#endif
     uint8_t sm = shared_state.pio_sm = config->pio_sm;
     pio_sm_claim(audio_pio, sm);
 
-    uint offset = pio_add_program(audio_pio, &audio_i2s_program);
+
+    const struct pio_program *program =
```

Comment:
There is some schizophrenia in the existing code about whether things should be #define or config variables. that said, I think we should just make the swapping a #define that can be set in the board header. I'm not aware of anyone using multiple different I2S

Reply:
what would you like the #define to be called? At least two alternatives occurred to me: `PICO_AUDIO_I2S_CLOCK_PINS_SWAPPED` and `PICO_AUDIO_I2S_CLOCK_PINS_BCLK_FIRST`?

I went ahead with PINS_SWAPPED but can change it if you like another alternative better.